chart/templates/traefikproxy/configmap.yaml

```diff
@@ -0,0 +1,95 @@
+{{- if .Values.traefikproxy.enabled -}}
+{{- if .Values.traefikproxy.defaultConfigmap -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "kyoo.traefikproxy.fullname" . }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "kyoo.labels" (dict "context" . "component" .Values.traefikproxy.name "name" .Values.traefikproxy.name) | nindent 4 }}
+  {{- with (mergeOverwrite (deepCopy .Values.global.configmapAnnotations) .Values.traefikproxy.configmapAnnotations) }}
+  annotations:
+    {{- range $key, $value := . }}
+    {{ $key }}: {{ $value | quote }}
+    {{- end }}
+  {{- end }}
+data:
+  dynamic_config.yaml: |
+    http:
+      routers:
+        basepath:
+          entryPoints:
+          - web
+          service: front
+          rule: "PathPrefix(`/`)"
+        video:
+          entryPoints:
+          - web
+          middlewares:
+          - phantomtoken
+          service: transcoder
+          rule: "PathPrefix(`/video`)"
+        auth:
+          entryPoints:
+          - web
+          service: auth
+          rule: "PathPrefix(`/auth/`)"
+        wellknown:
+          entryPoints:
+          - web
+          service: auth
+          rule: "PathPrefix(`/.well-known/`)"
+        api:
+          entryPoints:
+          - web
+          middlewares:
+          - phantomtoken
+          service: api
+          rule: "PathPrefix(`/api/`)"
+        swagger:
+          entryPoints:
+          - web
+          middlewares:
```

Comment:
The swagger shouldn't have the middleware, that's an overisight on my part (i'll fix it on the docker compose)

Reply:
#1147